src/components/Modals/ClaimNameFatFingerModal/ClaimNameFatFingerModal.types.ts

```diff
@@ -0,0 +1,16 @@
+import { Dispatch } from 'redux'
+import { ModalProps } from 'decentraland-dapps/dist/providers/ModalProvider/ModalProvider.types'
+import { claimNameRequest, ClaimNameRequestAction } from 'modules/ens/actions'
+
+export type Props = ModalProps & {
```

Comment:
I don't know if make sense here, but `metadata` type? maybe?

Reply:
I deleted it by accident! my bad